.github/workflows/static.yml

```diff
@@ -24,6 +24,7 @@ concurrency:
 jobs:
   # Single deploy job since we're just deploying
   deploy:
+    if: ((github.repository == vars.REPOSITORY) || (github.repository == 'bcgov/aries-oca-bundles'))

```

Comment:
This works perfectly fine, wondering if we want to use the GitHub repository owner though (see [this](https://github.com/hyperledger/aries-cloudagent-python/blob/main/.github/workflows/publish.yml#L46)) in case we decide to rename the repo in the future.